### ISSUE TITLE
New request: wiki.archlinux.org

### ISSUE THREAD

**Issue by vitaly-zdanevich** [openzim/zim-requests#1087]
- Website URL: wiki.archlinux.org
- License: **[GNU Free Documentation License 1.3 or later](https://www.gnu.org/copyleft/fdl.html)**
- Desired ZIM Title: **Arch Wiki**
- Desired ZIM Description: **A lot of useful information about Linux in general**
- Desired ZIM Icon https://wiki.archlinux.org/favicon.ico
- Language (ISO 639-3): **eng**
- Is this a MediaWiki?: **yes**

**Comment by benoit74** [comment]
Going to be solved soon, see https://github.com/openzim/zim-requests/issues/1287

**Comment by vitaly-zdanevich** [comment]
Looks like the same?

![Image](https://github.com/user-attachments/assets/944c5878-13b9-4f81-8088-1d6006fa55b2)

**Comment by benoit74** [comment]
I know, UI sucks, not my call. If you look at URL / file names, there is a `nopic` (no pictures) and a `maxi` (everything except video / audio) version. Not sure we need both for Arch wiki, but again, not my call, this has been decided before I looked at this recipe.

**Comment by Popolechien** [comment]
Yeah the nopic version was initially thought for Wikipedia, I think a common-sense approach would not hurt. Let's delete the `nopic` version for this one (I have already update the recipe). Requested at #1301
